detok/addfcodes.c

```diff
@@ -372,20 +369,25 @@ bool add_fcodes_from_list(char *vf_file_name)
 			bool found_spf = FALSE;
 			int indx;
 			for (indx = 0; indx < spcl_func_count; indx++) {
-				if ( strcmp( vs_fc_name, spcl_func_list[indx].name) == 0 ) {
+				if (strcmp
+				    (vs_fc_name,
```

Comment:
Is this (parens on the next line) really Linux style? And even if, is this something to emulate?